Mathlib/NumberTheory/ModularForms/JacobiTheta/OneVariable.lean

```diff
@@ -3,16 +3,14 @@ Copyright (c) 2023 David Loeffler. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: David Loeffler
 -/
-import Mathlib.Analysis.SpecialFunctions.Gaussian
-import Mathlib.Analysis.Complex.LocallyUniformLimit
-import Mathlib.Analysis.Complex.UpperHalfPlane.FunctionsBoundedAtInfty
-import Mathlib.Analysis.Complex.UpperHalfPlane.Topology
+import Mathlib.NumberTheory.ModularForms.JacobiTheta.TwoVariable
```

Comment:
Why does the one-variable file import the two-variable file? This seems counter-intuitive.

Reply:
OK, I see that some material from that file seems to be used further down.
Maybe one can collect what is needed for both in a "Basic" file and import that from both? But I haven't looked carefully at what needs what, so this may not be reasonable...

Reply:
I take it back :smile: I had to realize that the one-variable function is a special case of the two-variable function...